src/mixins/render-vega-lite/Definitions/Transform/Transforms/CrossSection2dDefinitionObject.js

```diff
@@ -183,11 +246,12 @@ export default class CrossSection2dDefinitionObject extends PropertiesDefinition
         type: "cross_section2d",
         coords: {
           x: this.x,
-          y: this.y,
-          z: this.z
+          y: this.y
         },
         xy_cross_section: this.crossSectionLine,
-        cross_section_dimension: this.crossSectionDimensionName
+        num_points_x: this.numPointsX,
```

Comment:
Do these need to be camel case now? Thought @simoneves had made that change on the BE side

Reply:
Fixed - @johallar ready for re-review